index.html

```diff
@@ -232,6 +232,9 @@ <h3>Easy to install</h3>
 # For CUDA 11.1
 pip install cupy-cuda111
 
+# For CUDA 11.2
+pip install cupy-cuda112
```

Comment:
Could you add a link to https://github.com/cupy/cupy/issues/4704?
```suggestion
pip install cupy-cuda112
(see <a href="https://github.com/cupy/cupy/issues/4704">here</a> for Linux)
```

Reply:
Applied the suggestion!